pandas/core/computation/eval.py

```diff
@@ -154,11 +158,20 @@ def _check_for_locals(expr, stack_level, parser):
         for toknum, tokval in tokenize_string(expr):
             if toknum == tokenize.OP and tokval == '@':
                 raise SyntaxError(msg)
-
-
-def eval(expr, parser='pandas', engine=None, truediv=True,
-         local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=False):
+    return None
+
+
+def eval(expr: str,
+         parser: str = 'pandas',
+         engine: str = None,
+         truediv: bool = True,
+         local_dict: List[Dict[str, Any]] = None,
+         global_dict: List[Dict[str, Any]] = None,
+         resolvers: Tuple = (),
```

Comment:
So does this only accept a Tuple of Dict? If so we want to add `Tuple[Dict[?, ?], ...]` as the annotation (replacing ? with appropriate types) and update the docstring which currently suggests a list is appropriate